app/controllers/api/v1/gossip_nodes_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class GossipNodesController < BaseController
+      def index
+        if nodes_params.has_key? "staked"
+          staked = ["true", true].include? nodes_params[:staked] ? true : false
+        else
+          staked = nil 
+        end
+        nodes = GossipNodeQuery.new(network: nodes_params[:network]).call(
+          staked: nodes_params[:staked] == "true",
```

Comment:
Wouldn't it be better to use boolean here?

Reply:
result of `nodes_params[:staked] == "true"` is a boolean, and I think this param is a string